documentation/assemblies/metrics/assembly-metrics.adoc

```diff
@@ -3,34 +3,38 @@
 // deploying/deploying.adoc
 
 [id='assembly-metrics-{context}']
-= Introducing Metrics to Kafka
+= Introducing metrics to Kafka
 
-This section describes how to monitor your Strimzi deployment.
+[role="_abstract"]
+You can use Prometheus and Grafana to monitor your Strimzi deployment.
 
 Depending on your requirements, you can:
 
-* xref:assembly-metrics-setup-{context}[Set up and deploy Prometheus and Grafana]
-* xref:assembly-kafka-exporter-{context}[Configure the `Kafka` resource to deploy Kafka Exporter with your Kafka cluster]
-+
-Kafka Exporter provides additional monitoring related to consumer lag.
+* xref:assembly-metrics-setup-{context}[Set up and deploy Prometheus to expose metrics]
```

Comment:
The order of these bullets doesn't match the order of the modules? I.e. the first module is about Kafka Exporter.